### PR TITLE
chore: upgrade Cypress to 6.2.1

### PR DESCRIPTION
**Pull request by ktmud**
### SUMMARY

Upgrade Cypress to 6.2.1 with some required refactoring:

1. Remove `readResponseBlob` since the new `cy.intercept(..)` API already reads JSON responses automatically.
2. Remove `verifyResponseCodes` since it's very unlikely that we get an incorrect status code yet still receive the correct response.
3. Add a [`FormData` parser](https://github.com/apache/superset/blob/f3d642a3920d40033be57bdf07f5269e02bc695a/superset-frontend/cypress-base/cypress/utils/parsePostForm.ts) since `request.body` is no longer a native `FormData` object
4. Replace a couple of `.should('not.visible')` to `.should('not.exist')` because Cypress will now throw errors for elements that do not exist.

Upgrading since I wanted to add more tests for time-series table and box plot viz.

Reference: [Cypress's official migration guide](https://docs.cypress.io/guides/references/migration-guide.html)

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A

### TEST PLAN

CI must pass

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/cypress-base/cypress/integration/explore/visualizations/table.test.ts [hunk 1]
```diff
@@ -98,7 +100,7 @@ describe('Visualization > Table', () => {
     });
     cy.verifySliceSuccess({
       waitAlias: '@getJson',
-      querySubstring: /groupby.*name/,
+      querySubstring: /group by.*name/i,
```
Comment (ktmud) [inline superset-frontend/cypress-base/cypress/integration/explore/visualizations/table.test.ts:103]: Previously this check also fails, but it doesn't report as an error.

File: superset-frontend/cypress-base/cypress/integration/dashboard/tabs.test.js [hunk 2]
```diff
@@ -164,67 +132,56 @@ describe('Dashboard tabs', () => {
...
-      const requestParams = JSON.parse(requestFormData.get('form_data'));
+    cy.wait('@treemapRequest').then(({ request }) => {
+      const requestBody = parsePostForm(request.body);
+      const requestParams = JSON.parse(requestBody.form_data);
       expect(requestParams.extra_filters[0]).deep.eq({
         col: 'region',
-        op: 'IN',
-        val: ['South Asia'],
+        op: '==',
+        val: 'South Asia',
```
Comment (villebro) [inline superset-frontend/cypress-base/cypress/integration/dashboard/tabs.test.js:149]: I'm surprised this (and the others below) even worked before?

**Review by villebro — APPROVED** [review]
OH wow, so much cleaner!

File: superset-frontend/cypress-base/cypress/integration/dashboard/tabs.test.js (hunk 2 above)
Comment (ktmud) [reply superset-frontend/cypress-base/cypress/integration/dashboard/tabs.test.js:147]: I think previously Cypress will just let this `expect` check in `wait.then(...)` pass even if it fails.

**Comment by jkoj52jkoj52** [comment]
Thank

**Pull request merged**